infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -598,20 +562,18 @@ StreamLog buildInMemoryStreamLog() {
             return new InMemoryStreamLog();
         }
 
-        StreamLog buildStreamLog(@Nonnull LogUnitServerConfig config,
-                                 @Nonnull ServerContext serverContext) {
-            return new StreamLogFiles(serverContext, config.isNoVerify());
+        StreamLog buildStreamLog(@Nonnull ServerContext serverContext) {
```

Comment:
There are two "god" objects in Corfu: CorfuRuntime and ServerContext. It would be great to try to avoid using them. 
As I mentioned earlier that's why I have created those small immutable config files like LogUnitServerConfig. It helps keep code flexible